.github/workflows/build.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         # The jApiCmp diff compares current to latest, which isn't appropriate for release branches, or for bot-generated PRs
         # this fails on windows because of the bash-specific if/then/else syntax, but that's ok
         # because we only need to run this validation once (on any platform)
-        if: ${{ matrix.os != 'windows-latest' && !startsWith(github.ref_name, 'release/') && !startsWith(github.base_ref, 'release/') && (github.actor != 'opentelemetrybot') }}
+        if: ${{ matrix.os != 'windows-latest' && !startsWith(github.ref_name, 'release/') && !startsWith(github.base_ref, 'release/') }}
```

Comment:
Is excluding the bot no longer necessary? If so, should update the comment: 

> or for bot-generated PRs
        # this fails on windows because of the bash-specific if/then/else syntax, but that's ok
        # because we only need to run this validation once (on any platform)

Reply:
Its possible it was a relic from when we didn't have post release automation. Let's remove it, update the comment, and wait and see!

Reply:
this breaks the version update process post-release, I believe. It's needed so that the that final PR doesn't break with the diff update.